packages/terra-pills/src/private/_PillsUtils.js

```diff
@@ -0,0 +1,49 @@
+const getPillWidth = (pill) => {
+  const pillWidth = pill.parentNode.getBoundingClientRect().width;
+  return pillWidth;
+};
+
+const handleRollUp = (pillListRef) => {
```

Comment:
Could we change this name to something that better describes what the function is doing? Maybe `getRollUpIndex`

Reply:
Related to my above comment: agree. The trade-off for my (desired, but I'm not the boss here) suggestion to not doc self-explaining functions, those functions _must_ have accurate names.

Reply:
One idea if this function is really a handler: create one wrapper handler function and wrap it around a more specifically named function. Something like:
```
# pseudocode for brevity:
def getRollUpIndex(args, kwargs):
   ... # do specific stuff
   return index

def handleClick(args, kwargs):
    return getRollUpIndex()
```

Reply:
added 40ee383faaf027a3fbbd63aec56cf8e88f282029